test/controllers/demo_group_controller_test.rb

```diff
@@ -76,15 +76,15 @@ class DemoGroupControllerTest < ActionDispatch::IntegrationTest
           end
 
           it 'should define render_authenticate_user_error' do
-            assert @controller.methods.include?(:render_authenticate_user_error)
+            assert @controller.methods.include?(:render_authenticate_error)
           end
 
           it 'should define render_authenticate_mang_error' do
-            assert @controller.methods.include?(:render_authenticate_mang_error)
+            assert @controller.methods.include?(:render_authenticate_error)
           end
 
           it 'should define render_authenticate_member_error' do
-            assert @controller.methods.include?(:render_authenticate_member_error)
+            assert @controller.methods.include?(:render_authenticate_error)
```

Comment:
These three tests now make less sense: their `it` descriptions talk about method names that are not checked for existence.